src/content/community/events/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Ethereum events
+description: How to get involved in the Ethereum community.
+sidebar: false
+lang: en
+---
+
+## Upcoming events {#events}
+
+**Every month, there are major Ethereum events around the world.** Consider attending one near you to meet more people in the community, learn about employment opportunities, and develop new skills.
+
+<UpcomingEventsList/>
```

Comment:
Wasn't noticing the UpcomingEventsList rendering. I think we talked about this already though, but if you need a hand let me know.

Reply:
We ran out of events!

- [Added events here](https://github.com/ethereum/ethereum-org-website/pull/4521/commits/1ceb4a31d71e8a1fa50a738f0db34931d820e00e)
- [Starter code for individual cards for each event](https://github.com/ethereum/ethereum-org-website/pull/4521/commits/dd9423be95871778336616a65ec828fe20ace945)

Reply:
Amongst other things would love if we could add some styling to connect cards akin to [how it is done here](https://bitcoin.org/en/events)

Reply:
We should have some fallback if there's 0 upcoming events, e.g. "We're not aware of any upcoming events. Know of one? Please add it!" with a link to create an issue.